src/private.rs

```diff
@@ -1,23 +1,24 @@
-use std::fmt;
-
-use super::{AbiEntry, AbiFunction, AbiMetadata, AbiRoot, RootSchema, SCHEMA_VERSION};
-
+use super::{
+    ensure_current_version, AbiBody, AbiFunction, AbiMetadata, AbiRoot, RootSchema, SCHEMA_VERSION,
+};
 use serde::{Deserialize, Serialize};
+use std::fmt;
 
 /// Core ABI information, with schema version and identity hash.
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct ChunkedAbiEntry {
```

Comment:
It's private, but should we rename this too?

Reply:
Hmm, I assumed "entry" here referred to an entry among other chunks and not an entry in ABI like `AbiEntry` was.

Reply:
Oh yeah, that's a good way to look at it. Ignore this then.